design/one-pager-scope-workload-interaction-mechanism.md

```diff
@@ -0,0 +1,141 @@
+# Scopes and workloads interaction mechanism in OAM
+
+* Owner: Ryan Zhang (@ryanzhang-oss)
+* Status: Implemented
```

Comment:
I think it's copy-paste mistake here.

Reply:
not exactly... I think Ryan came up with the proposal and I just follow to update codes.